src/conf_mode/system-ip.py

```diff
@@ -70,7 +70,7 @@ def apply(opt):
     sysctl('net.ipv4.fib_multipath_use_neigh', tmp)
 
     tmp = '0'
-    if isinstance(dict_search('multipath.ignore_unreachable_nexthops', opt), dict):
+    if isinstance(dict_search('multipath.layer4-hashing', opt), dict):
```

Comment:
This module uses `conf.get_config_dict(base, key_mangling=('-', '_'), get_first_key=True)` which means the resulting dictionary will never have a hyphen (-) within its keys. Please use `layer4_hashing` instead.

Thanks for the fix!

Reply:
Done.